public/js/components/Editor.js

```diff
@@ -77,6 +81,13 @@ const Editor = React.createClass({
     }
   },
 
+  resizeBreakpointGutter() {
```

Comment:
This code seems like it should live in `util/editor` as it is CM specific and does not deal with the business logic of debugging per-se